x-pack/plugins/integration_assistant/common/api/model/common_attributes.schema.yaml

```diff
@@ -36,6 +36,27 @@ components:
       items:
         type: object
 
+    SampleFormat:
+      type: object
+      description: Format of the provided samples.
+      required:
+        - name
+      properties:
+        name:
+          type: string
+          description: The name of the sample format.
+          enum:
+            - ndjson
+            - json
```

Comment:
Rather than modifying this object every time we append an enum , extract this enum out into own schema def and add `$ref` here.

[Example](https://github.com/elastic/kibana/pull/190588/files#diff-1e08a5b58a78fa8e84b3553e0167cca6912b93b70eac0156030712a86d0128a6R86-R103)

Reply:
Done